src/Uno.UI/UI/LayoutHelper.cs

```diff
@@ -175,6 +202,11 @@ internal static Size Subtract(this Size left, Size right)
 		[Pure]
 		internal static Size Subtract(this Size left, Thickness right)
 		{
+			if (right == Thickness.Empty)
```

Comment:
`Thickness.Empty` is not part of the WinUI API, I would instead also compare with `default` (Especially since `Size.Empty` is actually [-∞,-∞] and `Rect.Empty` is [∞, ∞, -∞, -∞] ... I'm not convinced that `Thickness.Empty` should be [0,0,0,0] as it is currently.)